valiu-node-commons/src/fixedu128.rs

```diff
@@ -0,0 +1,385 @@
+use core::{fmt, ops};
+use num_traits::{
+    identities::{One, Zero},
+    sign::Signed,
+    Bounded, Num,
+};
+use sp_arithmetic::{
+    traits::{CheckedAdd, CheckedDiv, CheckedMul, CheckedShl, CheckedSub, Saturating, Unsigned},
+    FixedPointNumber,
+};
+
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Default,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    parity_scale_codec::Decode,
+    parity_scale_codec::Encode,
+)]
+pub struct FixedU128(sp_arithmetic::FixedU128);
```

Comment:
`sp_arithmetic::FixedU128` doesn't have all necessary implementations for balance usage, e.g., `RemAssign`, `num_traits::One`, `DivAssign` and the list goes on. These requirements come from substrate itself and the orml crate

My search for this use-case was unsuccessful. Thanks for pointing out `https://github.com/encointer/substrate-fixed` and I hope it will be good enough